lfs/scanner.go

```diff
@@ -174,9 +184,8 @@ func ScanIndex() ([]*WrappedPointer, error) {
 // channel from which sha1 strings can be read.
 func revListShas(refLeft, refRight string, opt ScanRefsOptions) (chan string, error) {
 	refArgs := []string{"rev-list", "--objects"}
-	if opt.scanAll {
-		refArgs = append(refArgs, "--all")
-	} else {
+	switch opt.ScanMode {
+	case ScanTwoRefsDelta:
```

Comment:
Nitpick but it's not always scanning two refs, it can scan one (refRight == "") and in fact that's the more common usage. I'd call it ScanRefs or something similar instead.